yarn.lock

```diff
@@ -1,3510 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
My understanding is it's a best practice to have this file checked in (and the analogous Cargo.lock for the server-side stuff) for reproducible build results.

https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/